wellcomecollection/wellcomecollection.org
# Use string for a/v duration

For #11158 

## What does this change?

Changes Audio/Video duration field types from `number` to `text` – previously we were asking for a number of minutes and we'd calculate the `mm:ss` from that, but now we just ask for a string in the form `mm:ss`. We _could_ add a Prismic linting step to ensure entries are in the correct format, but maybe that's too much?

## How to test

Run locally using the `wellcomecollection-stage` repository endpoint, and with `toggle_prismicStage` set to `true` then visit the [audio](http://localhost:3000/guides/exhibitions/ZthrZRIAACQALvCC/audio-without-descriptions) and [video](http://localhost:3000/guides/exhibitions/ZthrZRIAACQALvCC/bsl) guide stop overview pages and check the durations render

## How can we measure success?

Easier for content editors to enter the correct durations

## Have we considered potential risks?

The audio/video content is edited without the duration being updated (although this was already an issue with the current implementation)
